webrtc.html

```diff
@@ -1755,6 +1762,36 @@ <h4>Set the RTCSessionDescription</h4>
                               </li>
                             </ol>
                           </li>
+                          <li>
+                            <p>If <var>description</var> is of type <code>"answer"</code>
+                            or <code>"pranswer"</code>, then run the following steps:</p>
+                            <ol>
+                              <li>
+                                <p>If the offer contained a request to send simulcast and
+                                the answer indicates that simulcast is not supported or
+                                desired by the answerer, then set the
```

Comment:
I'd like to remove "by the answerer" here, since we've avoided mentioning offerer/answerer thus far in this algorithm, and thus any outside inferences. We're in a for-loop here where the operands are *description* and *transceiver*.

Reply:
<var>description</var> covers it, so i will drop it.